src/app/pages/checkout/confirmation-dialog/confirmation-dialog.component.html

```diff
@@ -0,0 +1,6 @@
+<h1 mat-dialog-title>Confirm</h1>
+<div mat-dialog-content>{{confirmMessage}}</div>
+<div mat-dialog-actions>
+    <button mat-dialog-close>Cancel</button>
+    <button mat-dialog-close [mat-dialog-close]="true">Confirm</button>
```

Comment:
```suggestion
    <button mat-dialog-close>Confirm</button>
```
It should work without it, please check

Reply:
It won't work, because I check this value
![image](https://user-images.githubusercontent.com/53089462/104199629-553a9c80-5430-11eb-8892-612b8c44f748.png)